libs/feature/diary/src/lib/access-learning-diary.ts

```diff
@@ -75,12 +76,23 @@ async function updateDiaryDetails(username: string, id: string, endDate: Date) {
 			return;
 		}
 
+		const SIX_HOURS_IN_MS = 6 * 60 * 60 * 1000;
```

Comment:
hier gerne date-fns benutzen um magic numbers zu vermeiden. 

```suggestion

	const SIX_HOURS_MS = hoursToMilliseconds(6);
```